combat-trainer.lic

```diff
@@ -2970,6 +2973,10 @@ class GameState
     DRSpells.active_spells.include?('Call from Beyond')
   end
 
+  def cfb_recast?
```

Comment:
Any particular reason to make this part of GameState? It seems like its only used in LootProcess. Would it make more sense to define it there?